### DISCUSSION TITLE
ait-wiki/mechanics/tardis/chameleon/

### DISCUSSION THREAD

**Discussion by giscus** [amblelabs/ait-wiki#9] [General]
# ait-wiki/mechanics/tardis/chameleon/

TODO: IMAGE
Chameleon exterior also known as adaptive box is a special kind of exterior category.
When activated, it turns into a tree/desert well/chorus plant/giant mushroom and etc depending on the biome.
Due to the way it was implemented it has 2 qualities:
It’s undetectable. You can’t use F3, WAILA or X-ray mods to find a disguised TARDIS. It’s dynamic. It doesn’t have any presets, therefore it finds the shape to take dynamically.

https://pavatus.github.io/ait-wiki/mechanics/tardis/chameleon/

**Comment by evanwolfyou** [comment]
How do you get this?

**Reply by drtheodor** [reply]
Its in beta

**Comment by itzbnny** [comment]
We need to find a way to bring this feature back

**Reply by drtheodor** [reply]
so true
